dev/com.ibm.ws.jdbc_fat_derby/test-applications/trandriver/src/jdbc/tran/none/driver/TranNoneDataSource.java

```diff
@@ -47,6 +48,14 @@ public TranNoneDataSource() {
         impl = new EmbeddedDataSource();
     }
 
+    public void setServerName(String serverName) {
+        this.serverName = serverName;
```

Comment:
I think I misled you.  Instead you should remove the serverName variable and instead do impl.setServerName and impl.getServerName